queries/python/indents.scm

```diff
@@ -20,6 +16,19 @@
   (concatenated_string)
 ] @indent
 
+((list) @aligned_indent
+ (#set! "delimiter" "[]")
+ (#set! "dedent_bare_closing_delim" 1)
```

Comment:
You should be able to exclude `1` .

Reply:
Tried but that doesn't work - I'll have another shot at that, but I'm not sure how that's supposed to work as it's hard to distinguish between not set and set without setting something that evaluates in a truthy way